app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_comments_counter
+    comment_counter = post.comments.count
+    post.update(comments_counter: comment_counter)
```

Comment:
[OPTIONAL] Just as I pointed out in the post model, you could have just incremented the count and called the method that updates the count after saving or commiting